angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.component.ts

```diff
@@ -306,8 +306,8 @@ import {
             >
               {{ disclaimer.text }}
             </p>
+            <ng-content select="[additional-disclaimer-slot]"></ng-content>
```

Comment:
I'm glad we came up with this solution. This is a good workaround that gives our users what they need without requiring us to do the API refactor immediately.